package.json

```diff
@@ -7,7 +7,9 @@
     "test": "jest",
     "test:watch": "jest --watch",
     "test:coverage": "jest --coverage",
-    "prettier": "pretty-quick"
+    "prettier": "pretty-quick",
+    "add-contrib": "all-contributors add",
+    "generate-contribs": "all-contributors generate"
```

Comment:
For consistency with other tasks these could be `contributors:add` and `contributors:generate`, what do you think?

Reply:
Yea I was trying to shorten it - I'll stick to consistency 👍